autozone-poc/src/components/single-blog/Comments.js

```diff
@@ -0,0 +1,104 @@
+import React, { Fragment } from 'react';
+import '../single-blog/single-blog.css';
+import Button from '../../utlis/button/Button';
+
+const initialList = [];
+
+const Comments = () => {
+  const [value, setValue] = React.useState('');
```

Comment:
If possible merge all the state into a single state and handle the validation.

Reply:
Done